jbmc/src/java_bytecode/README.md

```diff
@@ -43,6 +43,45 @@ To be documented.
 
 To be documented.
 
+\section java-bytecode-remove-instanceof Remove instanceof
+
+Java has an operator called `instanceof`, which is used to check if an instance of a class is of a particular type. For example given that `A` and `B` are the identifiers of classes, which derive directly from Object. If the following example java code is executed -
```

Comment:
Lead with a one sentence summary of the whole thing before going into detail about the `instanceof` operator. I.e. 

```
\ref remove_instanceof.h removes the bytecode instruction `instanceof ` and replaces it with two instructions:
 - check whether the pointer is null
 - if not null, does the class identifier match the type of any of its subtypes
```